config/samples/resources/bigqueryconnectionconnection/aws-connection/bigqueryconnection_v1alpha1_bigqueryconnectionconnection.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: bigqueryconnection.cnrm.cloud.google.com/v1beta1
+kind: BigQueryConnectionConnection
+metadata:
+  name: bigqueryconnectionconnection-sample
+spec:
+  location: aws-us-east-1
+  projectRef:
+    external: ${projectId}
```

Comment:
I believe in samples this should be `${PROJECT_ID?}`. Have you tried to run the sample test and check if it works?

```
go test -v -tags=integration ./config/tests/samples/create/ -run-tests aws-connection
```

We usually also added a comment `# Replace ${PROJECT_ID?} with your project ID.` in the sample so that users are reminded to replace the project ID when they use the samples.

Reply:
Now it works! Thanks a lot